typescript/cli/src/context/context.ts

```diff
@@ -16,14 +16,18 @@ import {
 } from '@hyperlane-xyz/sdk';
 import { isHttpsUrl, isNullish, rootLogger } from '@hyperlane-xyz/utils';
 
+import { DEFAULT_STRATEGY_CONFIG_PATH } from '../commands/options.js';
 import { isSignCommand } from '../commands/signCommands.js';
+import { readChainSubmissionStrategyConfig } from '../config/strategy.js';
```

Comment:
Be careful to avoid import cycles. The commands use context so context shouldn't import from the commands

Reply:
You are right, any ideas on where I could put constants such as this?